components/chip/chip.vue

```diff
@@ -145,6 +145,14 @@ export default {
       type: [String, Array, Object],
       default: '',
     },
+
+    /**
+     * Additional class name for the span element.
+     */
+    parentClass: {
+      type: [String, Array, Object],
+      default: '',
+    },
```

Comment:
Didn't we decide to go with labelClass in this PR? https://github.com/dialpad/dialtone-vue/pull/642 should be the same here.

Reply:
It is not the same ticket.
First we will merge the previous PR, then we will merge this one and it will be fixed.